### ISSUE TITLE
A problem of training R50 model under VIPSeg dataset

### ISSUE THREAD

**Issue by xihuachen9804** [zhang-tao-whu/DVIS#9]
Hi! Your work is excellent!
I try to train DVIS_R50 model under VIPSeg dataset. 
Do  I need to finetune the segmenter by following "Training on a new dataset"?  Or I just need to train by following "Training" and use minvis_pretrained_weights.pth like "minvis_ovis_R50.pth"?
I‘m looking forward to your answer. Thanks!

**Comment by zhang-tao-whu** [comment]
Yes, you need to finetune the segmenter on the vipseg dataset. I will release the weights of the fine-tuned segmenter, please wait a few hours.

**Comment by zhang-tao-whu** [comment]
[weight.pth](https://pan.baidu.com/s/1JJRr06zXBTX5hyDdKEQzow), the password is 'dvis'.

**Comment by xihuachen9804** [comment]
> [weight.pth](https://pan.baidu.com/s/1JJRr06zXBTX5hyDdKEQzow), the password is 'dvis'.

Thank you so much for your sharing!

**Comment by zhang-tao-whu** [comment]
MinVIS_VIPSeg_R50.pth and MinVIS_VIPSeg_SwinL.pth are released. Please refer to [here](https://github.com/zhang-tao-whu/DVIS/blob/main/MODEL_ZOO.md).

**Comment by yuyuan2** [comment]
> MinVIS_VIPSeg_R50.pth and MinVIS_VIPSeg_SwinL.pth are released. Please refer to [here](https://github.com/zhang-tao-whu/DVIS/blob/main/MODEL_ZOO.md).

Hi, I have a question about testing result on VIPSeg, I inference with VIPSeg Swin-L weights but get a VPQ  value of around 52, which is much lower than 57.6. Could you pls tell me how to calculate the reported VPQ value? 

I also would like to know how did you calculate VPQthing and VPQstuff.

Thanks

**Comment by zhang-tao-whu** [comment]
Which model did you use to predict the results? The accuracy achieved by the prediction results of 57.6 VPQ is based on DVIS-Offline (SwinL).

**Comment by yuyuan2** [comment]
Hi, actually I used DVIS-Offline (SwinL) model provided in MODEL.ZOO, I would like to know if the 57.6 VPQ are based on test set?

**Comment by zhang-tao-whu** [comment]
The predicted results for the thing and stuff can be found in the vpq-final.txt file after using the evaluation script [eval_vpq_vspw.py](https://github.com/zhang-tao-whu/DVIS/blob/main/utils/eval_vpq_vspw.py) provided by VIPSeg.

**Comment by zhang-tao-whu** [comment]
The accuracy presented in the DVIS paper refers to the validation set, for the accuracy on the test set, please refer to our championship technical report at [vps report](https://arxiv.org/abs/2306.04091).

**Comment by yuyuan2** [comment]
Thanks much for your information!

**Comment by yuyuan2** [comment]
Hi, Zhang,

I have another question,

For VSPW dataset, which is the eval.py file to get mIoU performance?  Did you also report on val dataset?

Thanks

**Comment by zhang-tao-whu** [comment]
I haven't tested the accuracy of the result on VSPW dataset locally. I only submitted the results to the [competition server](https://codalab.lisn.upsaclay.fr/competitions/7869#learn_the_details). Maybe you can find the evaluation code in VSPW's GitHub repository.
